src/test/java/core/basesyntax/ParseDataServiceTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.ParseDataService;
+import core.basesyntax.service.impl.ParseDataServiceImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ParseDataServiceTest {
+    private static ParseDataService parseDataService;
+
+    @BeforeClass
+    public static void initialSetup() {
+        parseDataService = new ParseDataServiceImpl();
+    }
+
+    @Test
+    public void parseData_emptyDataFromFile_ok() {
+        List<String> dataFromFile = new ArrayList<>();
+        List<Fruit> result = parseDataService.parseData(dataFromFile);
+        List<Fruit> expected = new ArrayList<>();
+        Assert.assertEquals(result, expected);
+    }
```

Comment:
In most cases, you first put the expected value and then the actual, I think it's good practice, so it should be followed everywhere)